mypyc/test/test_run.py

```diff
@@ -67,8 +67,7 @@
 ]
 
 files.append("run-python37.test")
-if sys.version_info >= (3, 8):
-    files.append("run-python38.test")
+files.append("run-python38.test")
```

Comment:
Can't we just include these two in the list immediately above instead of appending them after the list has been constructed?